File: Csharp/S1002.cs
using System;

class S1002{
    class Person{
        private int x_pos;
        private int y_pos;
        private int target_distance;
        public Person(){
            this.x_pos = 0;
            this.y_pos = 0;
            this.target_distance = 0;
        }

        public int get_x_pos(){
            return x_pos;
        }
        public int get_y_pos(){
            return y_pos;
        }
        public int get_target_distance(){
            return target_distance;
        }
        public void set_all(int x, int y, int dist){
            x_pos = x;
            y_pos = y;
            target_distance = dist;
        }
    }
    public static void Main(){
        Person p1 = new Person(), p2 = new Person();
        int loopControl = int.Parse(Console.ReadLine());

        while(loopControl-->0)
        {
            string[] input = Console.ReadLine().Split();
            int[] datas = Array.ConvertAll<string, int>(input, int.Parse);
            p1.set_all(datas[0],datas[1],datas[2]);
            p2.set_all(datas[3],datas[4],datas[5]);
            double distance = Math.Sqrt(Math.Pow(p2.get_x_pos()-p1.get_x_pos(),2)+Math.Pow(p2.get_y_pos()-p1.get_y_pos(),2));
            int r_sum = p2.get_target_distance() + p1.get_target_distance();
            int r_sub = Math.Abs(p2.get_target_distance() - p1.get_target_distance());
            
            if(distance == 0)
            {
                if(p1.get_target_distance() == p2.get_target_distance())
                {
                    Console.WriteLine("-1");
                }
                else
                {
                    Console.WriteLine("0");
                }
            }
            else if(distance == r_sum || distance == r_sub)
            {
                Console.WriteLine("1");
            }
            else if(distance < r_sum && distance > r_sub)
            {
                Console.WriteLine("2");
            }
            else if(distance > r_sum || distance < r_sub)
            {
                Console.WriteLine("0");
            }
        }
    }
}

File: Python/S1002.py
class Person:
    def __init__(self):
        self.x = 0
        self.y = 0
        self.target_distance = 0

import math

class Main:
    p1, p2 = Person(), Person()
    T = int(input())
    for _ in range(T):
        p1.x, p1.y, p1.target_distance, p2.x, p2.y, p2.target_distance = map(int,input().split())
        dist = math.sqrt(math.pow(p2.x-p1.x,2)+math.pow(p2.y-p1.y,2))
        r_sum, r_sub = p2.target_distance + p1.target_distance, abs(p2.target_distance - p1.target_distance)
        if dist == 0:
            if p1.target_distance == p2.target_distance:
                print("-1", sep='\n')
            else:
                print("0", sep='\n')
        elif dist == r_sum or dist == r_sub:
            print("1", sep='\n')
        elif dist < r_sum and dist > r_sub:
            print("2", sep='\n')
        elif dist > r_sum or dist < r_sub:
            print("0", sep='\n')